nibabies/workflows/bold/resampling.py

```diff
@@ -135,48 +199,422 @@ def select_target(subject_id, space):
         name="outputnode",
     )
 
-    # fmt: off
+    # 0, 1 = wm; 2, 3 = pial; 6, 7 = mid
+    # note that order of lh / rh within each surf type is not guaranteed due to use
+    # of unsorted glob by FreeSurferSource prior, but we can do a sort
+    # to ensure consistent ordering
+    select_wm = pe.Node(
+        niu.Select(index=[0, 1]),
+        name="select_wm",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
```

Comment:
Generally, we can run most nipype utility interfaces on the master process since they are not intense operations - no need to potentially submit to a job queue.

```suggestion
        mem_gb=DEFAULT_MEMORY_MIN_GB,
        run_without_submitting=True,
    )
```